app/src/controllers/administrate/events/reducer.js

```diff
@@ -15,13 +15,17 @@
  */
 
 import { combineReducers } from 'redux';
+import { createPurifyPageReducer } from 'common/utils/store';
 import { fetchReducer } from 'controllers/fetch';
 import { paginationReducer } from 'controllers/pagination';
 import { loadingReducer } from 'controllers/loading';
+import { PROJECT_DETAILS_PAGE } from 'controllers/pages';
 import { NAMESPACE } from './constants';
 
-export const eventsReducer = combineReducers({
+const reducer = combineReducers({
   events: fetchReducer(NAMESPACE, { contentPath: 'content' }),
   pagination: paginationReducer(NAMESPACE),
   loading: loadingReducer(NAMESPACE),
 });
+
+export const eventsReducer = createPurifyPageReducer(reducer, [PROJECT_DETAILS_PAGE]);
```

Comment:
Let's be more consistent with other files:
`(reducer, PROJECT_DETAILS_PAGE)`

Reply:
Fixed